javascript.js

```diff
@@ -0,0 +1,97 @@
+var game = document.getElementById("game");
+ var goo = 0;
```

Comment:
I think this variable indicates whether the game is over? But I'm not sure what the word goo means in this context. I'm sorry if it is a word from another language, it is ok to program using words from other languages.  But in general, we want to think about what will make sense to the developers who read our code.